docker-local/traefik/etc/traefik/traefik.yaml

```diff
@@ -0,0 +1,20 @@
+entryPoints:
+  HttpEntrypoint:
+    address: ":80"
+  HttpsEntrypoint:
+    address: ":443"
+  DrpcEntrypoint:
+    address: ":6666/tcp"
+  DrpcsEntrypoint:
+    address: ":7777/tcp"
```

Comment:
Should I duplicate the authservice ports here? So also make it 13000, 13001, 13002, 13003 like described in confluence.

Reply:
we're using the 20000 range now and defaults have already been updated in code: https://github.com/storj/gateway-mt/commit/7a9fd39dd5252e18c646922deee3817535f2378f

i'll update the confluence page since it's out of date :grimacing:

Reply:
This confluence page says "Internal Service Ports".

I am not sure the 1300x port should be applied here since a proxy is not the same internal service as the services which it proxies too.

@nergdron I see you made this page, how should we think about this?

Reply:
(in any case I don't think it has to be blocking as this is just development environment which can be changed easily)

Reply:
agreed, it probably shouldn't be made here